### PR TITLE
FI-1632: Update terminology for US Core 5

### PR DESCRIPTION
**Pull request by Jammjammjamm**

### CODE REVIEW

File: resources/value_sets.yml [hunk 1]
```diff
@@ -1717,3 +1717,121 @@
...
+  :system: http://hl7.org/fhir/ValueSet/medication-as-needed-reason
+  :path: asNeeded
+- :type: CodeableConcept
+  :strength: example
+  :system: http://hl7.org/fhir/ValueSet/participant-role
+  :path: performerType
+- :type: CodeableConcept
+  :strength: example
+  :system: http://terminology.hl7.org/ValueSet/v3-ServiceDeliveryLocationRoleType
+  :path: locationCode
```
Comment (arscan) [inline resources/value_sets.yml:1837]: I assume whatever context this is being used in knows what Resource type is relevant so you don't need to technically print out the Resource type here?  Might be nice to just put it in here if you have it at generation time so this is a more useful human-readable artifact for reviews too.  Just an idea -- I might be misinterpreting what this file is for.

Comment (Jammjammjamm) [reply resources/value_sets.yml:1837]: This file is used to check that all of the value sets we expect to be loaded are loaded. Adding the resource type would be easy if the file weren't made by hand.

File: lib/inferno/terminology/expected_manifest.yml [hunk 2]
```diff
@@ -1182,13 +1320,28 @@
...
   :file: terminology_hl7_org_CodeSystem_medicationrequest-category.msgpack
   :count: 4
   :type: bloom
   :code_systems: http://terminology.hl7.org/CodeSystem/medicationrequest-category
-- :url: urn:oid:2.16.840.1.113883.6.101
-  :file: urn_oid_2_16_840_1_113883_6_101.msgpack
-  :count: 970
+- :url: http://www.nubc.org/patient-discharge
+  :file: www_nubc_org_patient-discharge.msgpack
+  :count: 0
```
[the remark this answers is not in the document]
Comment (Jammjammjamm) [reply lib/inferno/terminology/expected_manifest.yml:1335]: This is the new system with licensing restrictions, so we're not actually loading it. It isn't used in a required binding, so we don't actually validate against it. This file doesn't actually exist, though, so this entry is misleading. We may need a special case to handle this.

File: lib/inferno/terminology/expected_manifest.yml [hunk 3]
```diff
@@ -678,15 +696,23 @@
...
 - :url: http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1010.6
   :file: cts_nlm_nih_gov_fhir_ValueSet_2_16_840_1_113762_1_4_1010_6.msgpack
-  :count: 172
+  :count: 196
   :type: bloom
   :code_systems:
   - http://hl7.org/fhir/sid/cvx
+- :url: http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1010.4
+  :file: cts_nlm_nih_gov_fhir_ValueSet_2_16_840_1_113762_1_4_1010_4.msgpack
+  :count: 1000
```
Comment (arscan) [inline lib/inferno/terminology/expected_manifest.yml:712]: This is an oddly round number, not a cap?

File: resources/value_sets.yml (hunk 1 above)
Comment (arscan) [reply resources/value_sets.yml:1837]: Oh, this file is made by hand?  Why do we need `path` then?

Comment (Jammjammjamm) [reply resources/value_sets.yml:1837]: It just comes straight from the US Core metadata: https://github.com/inferno-framework/us-core-test-kit/blob/main/lib/us_core_test_kit/generated/v3.1.1/metadata.yml#L93

File: lib/inferno/terminology/expected_manifest.yml (hunk 3 above)
Comment (Jammjammjamm) [reply lib/inferno/terminology/expected_manifest.yml:712]: It's not a cap. We have plenty of larger value sets.

File: lib/inferno/terminology/expected_manifest.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (Jammjammjamm) [reply lib/inferno/terminology/expected_manifest.yml]: I made an update so that this won't appear in the manifest.

**Review by arscan — APPROVED** [review]

**Pull request merged**